src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const arrayVersionsOfState = [];
```

Comment:
Avoid using data structure names in variable names, it's really confusing

Reply:
Got you, done!